src/mod/applications/mod_commands/mod_commands.c

```diff
@@ -3865,10 +3866,10 @@ SWITCH_STANDARD_API(uuid_answer_function)
 		if (status == SWITCH_STATUS_SUCCESS) {
 			stream->write_function(stream, "+OK\n");
 		} else {
-			stream->write_function(stream, "-ERROR\n");
+			stream->write_function(stream, "-ERR invalid uuid\n");
```

Comment:
The "invalid uuid" message is inaccurate, since switch_core_session_locate() would return a valid session for the uuid.  I think "cannot answer" is better.

From switch_channel_perform_answer() : 
``` 
if (channel->hangup_cause || channel->state >= CS_HANGUP) {
           return SWITCH_STATUS_FALSE;
 }
```

Reply:
good suggestion